projects/sbb-esta/angular-public/src/styles/typography/_form.scss

```diff
@@ -78,11 +80,13 @@
   @include publicOnly() {
     background: url($selectNativeBackground) 97.5% no-repeat;
```

Comment:
The padding-right change displaced the position of the arrow.
This can be solved by removing the percentage from background (and moving it ouside publicOnly/businessOnly) and adding `background-position: center right $selectArrowRight` (also replace `background-position-x` from the 4k/5k with `background-position: toPx($selectArrowRight * $scalingFactor4k/5k)`)